packages/lit-dev-content/samples/tutorials/pointer-events/01/before/my-element.ts

```diff
@@ -0,0 +1,35 @@
+import {LitElement, html, css} from 'lit';
+import {customElement, state} from 'lit/decorators.js';
+
+@customElement('my-element')
+class MyElement extends LitElement {
+  @state() log = '';
+
+  static styles = [css`
```

Comment:
I'd suggest some playground folds here to hide things like styling and focus the reader's attention on the parts that matter